docs/django-apps.md

```diff
@@ -0,0 +1,67 @@
+# Using Caluma as django apps
+
+## Disclaimer
+
+It is highly recommended to use Caluma as a dedicated service. However, there are usecases, where
+it might make sense to integrate Caluma into another django project.
+
+If you just want to get Caluma up and running, please see the documentation about [setting up
+the Caluma service](configuration.md).
+
+## Installation
+
+Caluma is on PyPI, so you can just
+
+```shell
+pip install caluma
+```
+
+
+## Configuration
+
+### Installed apps
+
+Add the caluma apps you want to use to your `INSTALLED_APPS`:
+
```

Comment:
Some note on the caluma-internal dependencies would be useful:

* caluma_workflow needs caluma_form to work correctly (as cases and work items point to documents)
* core should always be installed when using caluma (though, *technically*, it would work without it, as core doesn't contain any models
* caluma_user should always be added (same caveat as with core)

I would also highlight using `psqlextra`, as Caluma won't work without it (eg. with the standard postgresql backend)